x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/policies/package_policies.tsx

```diff
@@ -213,16 +218,6 @@ export const PackagePoliciesPage = ({ name, version }: PackagePoliciesPanelProps
           defaultMessage: 'Version',
         }),
         render(_version, { agentPolicy, packagePolicy }) {
-          const updatableIntegrationRecord = updatableIntegrations.get(
-            packagePolicy.package?.name ?? ''
-          );
-
-          const hasUpgrade =
```

Comment:
InMemoryPackagePolicy has a hasUpgrade property so we don't need to duplicate the logic here